.github/workflows/pages.yaml

```diff
@@ -39,15 +39,8 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip
-          pip install autodocsumm
-          pip install sphinx_rtd_theme
-          pip install sphinx_gallery
-          pip install sphinxcontrib-inlinesyntaxhighlight
-          pip install sphinx_toolbox
-          pip install numpy
-          pip install jinja2
-          pip install torch
+          python3.9 -m pip install --upgrade pip
+          python3.9 -m pip install numpy autodocsumm 'sphinx<6' sphinx_rtd_theme sphinx_gallery sphinxcontrib-inlinesyntaxhighlight sphinx_toolbox jinja2 torch
           cd python
           python setup.py develop
```

Comment:
Here we use both `python3.9` (line 43) and `python` (line 45). For consistency it is better to use one or another. Looks like `python-version: ["3.9"]` statement maps `python` to `python3.9`. We probably can simply use `python` everywhere in this file. 
The same in docs.yaml